flake.nix

```diff
@@ -71,13 +71,16 @@
             config.treefmt.build.devShell
             self'.devShells.rust
           ];
-          packages = with pkgs; [
+          packages = with pkgs; ([
             just
             nixci
             # For when we start using Tauri
             cargo-tauri
             trunk
-          ];
+          ]) ++ lib.optionals pkgs.stdenv.isDarwin (
+            [ pkgs.darwin.apple_sdk_11_0.frameworks.IOKit
+              pkgs.darwin.apple_sdk_11_0.frameworks.Security
+            ]);
```

Comment:
This should go in rust.nix (`rustBuildInputs`). Also, CI fails due to not having run autoformatter.

Reply:
Adding in `rustBuildInputs` is causing nix run failing. Updated and added to `devShells.rust`

Reply:
> causing nix run failing

Please post the full error.

---

devShells.rust already uses rustBuildInputs transitively, so the devShell will now have duplicate IOKit and Security frameworks.

Reply:
`error occurred: Command env -u IPHONEOS_DEPLOYMENT_TARGET "clang" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "--target=x86_64-apple-darwin" "-Wall" "-Wextra" "-o" "/private/tmp/nix-build-nix-browser-deps-0.1.0.drv-0/source/target/release/build/tao-fde661b9405e71cb/out/src/platform_impl/macos/carbon_hotkey/carbon_hotkey_binding.o" "-c" "src/platform_impl/macos/carbon_hotkey/carbon_hotkey_binding.c" with args "clang" did not execute successfully (status code exit status: 1).`